src/goto-symex/goto_symex_state.cpp

```diff
@@ -288,8 +289,7 @@ static void assert_l1_renaming(const exprt &expr)
   #if 1
   if(check_renaming_l1(expr))
   {
-    std::cerr << expr.pretty() << '\n';
-    UNREACHABLE;
+    DATA_INVARIANT(false, "l1 renaming failed:\n" + expr.pretty());
   }
```

Comment:
I don't think this is a DATA_INVARIANT as it is not something that should always be true of an expr.  INVARIANT perhaps.

Reply:
CHECK_RETURN ?